app/src/androidTest/java/net/skyscanner/backpack/compose/dialog/BpkDialogTest.kt

```diff
@@ -115,6 +116,15 @@ class BpkDialogTest : BpkSnapshotTest() {
     }
   }
 
+  @Test
+  fun image() {
```

Comment:
can you please add a test variant for both left & right alignment for RTL & default variant please?